docker-gen.go

```diff
@@ -58,6 +58,13 @@ type Volume struct {
 	ReadWrite bool
 }
 
+type Mount struct {
+	Source      string
+	Destination string
+	Mode        string
+	RW          bool
```

Comment:
docker-gen exposes it's own API that I'd like to keep consistent.  It's not the same as go-dockerclient even though it uses it.